src/main/resources/application.properties

```diff
@@ -2,9 +2,9 @@ logging.level.org.springframework=ERROR
 
 ## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
 ## will eventually switch to cloud and have auth
-spring.datasource.url=jdbc:postgresql://localhost:5431/safeHomeDB_v0.01
-spring.datasource.username = postgres
-spring.datasource.password= 1298
+spring.datasource.url=jdbc:postgresql://ec2-54-197-234-117.compute-1.amazonaws.com:5432/d33f674k0jj8nk
+spring.datasource.username = cqerjydhappapg
+spring.datasource.password= 6f9a4baba08e8e8acd84a1b869fc61876bd7bf4934ad2e44d40d6e92ff2ec9b6
```

Comment:
Hey! thanks for updating the configuration. Since the project is public, I think we should export some of the credentials into an environment variable to prevent anyone who views the source from accessing the database. 

For example: 
`spring.datasource.username= ${SHOME_POSTGRES_USER}`

where we will share the configuration for SHOME_POSTGRES_USER amongst us privately.